src/helpers.js

```diff
@@ -39,7 +39,8 @@ export const replaceNodeAtPos = (position, content) => tr => {
   const $pos = tr.doc.resolve(before);
   if (canReplace($pos, content)) {
     tr = tr.replaceWith(before, before + node.nodeSize, content);
-    return cloneTr(setTextSelection(tr.selection.$from.pos - 1, -1)(tr));
+    const start = tr.selection.$from.pos - 1;
+    return cloneTr(setTextSelection(start >= 0 ? start : 0, -1)(tr));
```

Comment:
the actual fix

Reply:
```js
return cloneTr(setTextSelection(Math.max(start, 0), -1)(tr));
```

Reply:
yeah I though about it too, wasn't sure if it'd make any difference 🙃I'll update that